integration_tests/test_trash_list.py

```diff
@@ -30,22 +30,6 @@ def sort_lines(lines):
 
 class Test_describe_trash_list(Setup):
 
-    def test_should_output_the_help_message(self):
-
-        self.user.run_trash_list('--help')
-
-        assert_equals_with_unidiff(dedent("""\
-            Usage: trash-list [OPTIONS...]
-
-            List trashed files
-
-            Options:
-              --version   show program's version number and exit
-              -h, --help  show this help message and exit
-
-            Report bugs to https://github.com/andreafrancia/trash-cli/issues
-        """), self.user.output())
-
```

Comment:
This is not "fixing a test", this is "deleting test".

Reply:
Apology for the imprecise commit message (I meant it more like fixing the error generated during ci test)... However, I don't think there's any meaningful test to be done when the help message is generated by `argparse`...